File: samples_generated_code/sample_3/phi4/baseline_code_py/code_row_63.py
import sys

if len(sys.argv) != 2:
    print("Error: Please provide exactly one argument.")
    sys.exit(1)

input_argument = sys.argv[1]
print(input_argument)
trimmed_argument = input_argument.rstrip()
print(trimmed_argument)
sys.exit(0)

File: samples_generated_code/sample_2/qwen/baseline_code_c/code_row_63.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        printf("Error: Please provide exactly one argument.\n");
        exit(EXIT_FAILURE);
    }
    char *input = argv[1];
    printf("Original input: %s\n", input);
    while (*input && isspace((unsigned char)*input)) input++;
    char *end = input + strlen(input) - 1;
    while (end > input && isspace((unsigned char)*end)) end--;
    *(end + 1) = '\0';
    printf("Trimmed input: %s\n", input);
    exit(EXIT_SUCCESS);
}